web_tree_image/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :alt: License: AGPL-3
+
+=====================================
+Display images and icons in tree view
+=====================================
+
+This module defines a tree image widget, to be used with either binary fields
+or (function) fields of type character. Use ``widget='image'`` in your view
+definition. Optionally, set a ``height`` attribute. Default height is 16px.
+
+If you use the widget with a character field, the content of the field can be
+any of the following:
+
+* The absolute or relative location of an image. For example,
+  "/<module>/static/src/img/youricon.png"
+
+* A standard icon from the web distribution, without path or extension, For
+  example, 'gtk-open'
+
+* A dynamic image in a data url base 64 format. Prefix with
+  'data:image/png;base64,'
+
+Usage
+=====
+
+This module defines a new widget type for tree views columns.
+
+Set the attribute ``widget=image`` in a ``field`` tag in a tree view.
+You can also set ``height=<height>`` to set the height the image will have.
+Note that this just sets the CSS ``max-height`` attribute,
```

Comment:
Is this still the case without the Javascript?

Reply:
Yes @tarteo, In  V11 added a widget for an image but height and width are not proper, so no need to Javascript for particular widget. Only CSS we have a used and added a width in [here](https://github.com/OCA/web/pull/1006/files#diff-72b8546f14fd85b2b3b54f8eaca8c08bR1)

And those things are described in already https://github.com/OCA/web/pull/1006#issuecomment-409183102.

Thanks.